Generals/Code/GameEngine/Source/GameLogic/Object/Contain/MobNexusContain.cpp

```diff
@@ -347,8 +347,8 @@ UpdateSleepTime MobNexusContain::update()
 			{
 				ContainedItemsList::const_iterator it;
 				it = items->begin();
```

Comment:
While at it, perhaps make the declaration and assignment a single line? Looks a bit unholy right now.